src/appkit.cs

```diff
@@ -6094,6 +6094,7 @@ interface NSFontCollection : NSSecureCoding, NSMutableCopying {
 
 	[Lion]
 	[BaseType (typeof (NSFontCollection))]
+	[DisableDefaultCtor]
```

Comment:
So would that mean that Apple broke compat as well? They are the ones who did this I assume:

[FAIL] Default constructor not allowed for AppKit.NSMutableFontCollection : SIGILL

(then we crash).